### ISSUE TITLE
Review Uptime

### ISSUE THREAD

**Issue by sbordet** [eclipse/jetty.project#3515]
`Uptime` is currently a static singleton with a nested interface and implementation that are apparently not used.

Furthermore, it reports the uptime of the JVM while it is mostly used to report the `Server` uptime.

In the case of Server restarts, for example, it won't report the restarted `Server` uptime.

Why does this class need to support `Android`?

**Comment by joakime** [comment]
See: https://bugs.eclipse.org/443652

Android (and also JRE on compact3) does not have `java.lang.management.ManagementFactory`.

**Comment by sbordet** [comment]
I don't think we support Jetty (the server) on Android. Do we still need to support compact3?

**Comment by joakime** [comment]
We do support Jetty (the server) on Android v11+.
We should ask `jetty-users` (and `jetty-dev`) if compact3 is still needed.
